automerge-cli/src/error.rs

```diff
@@ -0,0 +1,17 @@
+use std::error::Error;
+use std::fmt;
+
+#[derive(Debug, PartialEq)]
+pub enum AutomergeCliError {
```

Comment:
It's probably worth using `thiserror` to derive the  Error trait here, it takes away most of the pain of generating nice error types and it's already a dependency of the `automerge-protocol` trait.

Reply:
👀 

Great! I'll take a look there, thanks for pointing me in the right direction.